### PR TITLE
Omit provenance field

### PR DESCRIPTION
**Pull request by guillemcordoba**

### CODE REVIEW

**Comment by jost-s** [comment]
@guillemcordoba How about a simple field `AppAgentWebsocket.agentPubKey` that becomes part of the constructor?

**Comment by guillemcordoba** [comment]
@jost Hum but then how is an app supposed discover the agentpubkey and pass that in? By calling `appWebsocket.appInfo()`?

And also, when agentpubkey is included in the result of `AppInfo`, doesn't this constructor argument become a foot gun?

File: README.md [hunk 1]
```diff
@@ -104,6 +103,7 @@ You need `holochain` and `hc` on your path, best to get them from nix with `nix-
 To perform the pre-requisite DNA compilation steps, and run the Nodejs test, run:
 ```bash
 nix-shell
+npm run build
```
Comment (jost-s) [inline README.md]: Why is building required before running the tests?

File: src/types.ts [hunk 2]
```diff
@@ -9,6 +9,16 @@ export type AnyDhtHash = HoloHash;
 export type KitsuneAgent = Uint8Array;
 export type KitsuneSpace = Uint8Array;
 
+/** Base64 hash types */
+
+export type HoloHashB64 = string;
```
Comment (jost-s) [inline src/types.ts:14]: This one is in `src/api/admin/types.ts` too. Please remove it there.

File: src/utils/base64.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { Base64 } from "js-base64";
+import { HoloHash, HoloHashB64 } from "../types.js";
+
+export function deserializeHash(hash: HoloHashB64): HoloHash {
```
Comment (jost-s) [inline src/utils/base64.ts]: Can we rename these to `encodeHashTo64` and `decodeHashFromB64` or something similar? It has nothing to do with serialization.

File: README.md (hunk 1 above)
Comment (guillemcordoba) [reply README.md]: Oh sorry it isn't? My bad, will remove

File: src/types.ts (hunk 2 above)
Comment (guillemcordoba) [reply src/types.ts:14]: Hum I think I removed it? I don't see it in the code...

Comment (jost-s) [reply src/types.ts:14]: Sorry, I looked at the wrong branch.

**Review by jost-s — APPROVED** [review]

**Pull request merged**